semaphore_admission.go

```diff
@@ -0,0 +1,65 @@
+package frankenphp
```

Comment:
`admission.go` ?

Please don't use `_` in file names as it's special build-tag syntax.